docs/config.rst

```diff
@@ -1,11 +1,606 @@
+-------------------
+
+Flow configurations are defined under the ``flows`` section of your ``cumulusci.yml`` file. Each flow configuration defines a flow which can be run using the ``cci flow run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
+
+Add a Custom Flow
+^^^^^^^^^^^^^^^^^
+
+To define a new flow for your project, simply add the name of the new flow under the ``flows`` section of your ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    greet_and_sleep:
+        group: projectName
+        description: Greets the user and then sleeps for 5 seconds.
+        steps:
+            1:
+                task: command
+                options:
+                    command: echo 'Hello there!' 
+            2:
+                task: util_sleep
+
+This is a flow comprised of two tasks: ``command`` greets the user by echoing a string, and ``util_sleep`` then tells CumulusCI to sleep for five seconds. You can reference how the flows are defined in the `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file.
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+
+If you want to add a step to a flow, use ``cci flow info <name>`` first to see the existing steps.
+
+    Example: ``dev_org``
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All *non-negative numbers and decimals* are valid as step numbers in a flow.
+You can add steps before, in-between, or after existing flow steps.
+If you want to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1. Example values would include 0, 0.3, and 0.89334. All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
```

Comment:
```suggestion
To add a step at the beginning of the dev org flow, any step number greater than or equal to zero and less than 1 is valid. For example, 0, 0.3, and 0.89334 all specify that the step execute before step 1 in the ``dev_org`` flow.
```
Heavens, three modal verbs in one paragraph is surely a record. Gotta watch out for these. https://confluence.internal.salesforce.com/display/DOCTEAM/modal+verbs

Reply:
Sorry, didn't realize "would" was on the list. Noted.